do.sh

```diff
@@ -17,4 +21,17 @@ if ! grep -q "set editing-mode vi" "${HOME}/.inputrc" 2>/dev/null; then
 fi
 
 
+echo "Configuring iptables firewall..."
```

Comment:
I just wanted to let you know that I need more attention. iptables are dangerous. , you can ban yourself forever with an invalid iptables rule.

Reply:
Sure